gradle-diff-task-dependencies/README.md

```diff
@@ -0,0 +1,47 @@
+Gradle Diff Dependency Task
```

Comment:
The project name, as much as the package and gradle plugin name, suck.
I am open to rename the plugin and main packages/classes to something clearer.